<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkfFilterItem</class>
 <widget class="QWidget" name="WkfFilterItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>79</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>80</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="typeComboBox">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Type</string>
        </property>
        <item>
         <property name="text">
          <string>Condition</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>And</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Or</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="enableCheckBox">
        <property name="toolTip">
         <string>Enabled</string>
        </property>
        <property name="text">
         <string>Enabled</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="invertCheckBox">
        <property name="toolTip">
         <string>Invert result</string>
        </property>
        <property name="text">
         <string>Invert</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="columnLineEdit">
        <property name="toolTip">
         <string>Column title</string>
        </property>
        <property name="placeholderText">
         <string>Column title...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="columnLabel">
        <property name="toolTip">
         <string>Column index</string>
        </property>
        <property name="text">
         <string>(col #)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="testComboBox">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Test</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="valueLineEdit">
        <property name="toolTip">
         <string>Value</string>
        </property>
        <property name="placeholderText">
         <string>Compares to...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="addButton">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Add child</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="add">
          <normaloff>../../../../../../../../.designer/backup</normaloff>../../../../../../../../.designer/backup</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="removeButton">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Remove</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="remove">
          <normaloff>../../../../../../../../.designer/backup</normaloff>../../../../../../../../.designer/backup</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="expandCheckBox">
     <property name="toolTip">
      <string>Expand children</string>
     </property>
     <property name="text">
      <string>Expand</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="subFiltersList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <layout class="QVBoxLayout" name="listLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>typeComboBox</tabstop>
  <tabstop>enableCheckBox</tabstop>
  <tabstop>invertCheckBox</tabstop>
  <tabstop>columnLineEdit</tabstop>
  <tabstop>testComboBox</tabstop>
  <tabstop>valueLineEdit</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>subFiltersList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
